packages/yourloops/pages/hcp/team-edit-dialog.tsx

```diff
@@ -260,14 +262,13 @@ function TeamEditDialog(props: TeamEditModalProps): JSX.Element {
         <strong>{modalTitle}</strong>
       </DialogTitle>
 
-      {infoLine}
+      {infoLine ?? <Box />}
```

Comment:
It was fine before, no?

Reply:
Another problem with `DialogTitle`, if used with a `dialogContent` just after, this one loss its padding top, so the content is partially covered by the title...